cli/vm/cli.go

```diff
@@ -616,7 +616,7 @@ func getInstructionParameter(c *cli.Context) (int, error) {
 	}
 	n, err := strconv.Atoi(args[0])
 	if err != nil {
-		return 0, fmt.Errorf("%w: %s", ErrInvalidParameter, err) //nolint:errorlint // errorlint: non-wrapping format verb for fmt.Errorf. Use `%w` to format errors
+		return 0, fmt.Errorf("%w: %s", ErrInvalidParameter, err)
```

Comment:
But it should be `%w: %w` then, I think.

Reply:
It'd be better to convert them to `%w: %w` in the same commit with comment removal, otherwise linter error will be present after removal but before double-%w.

Reply:
Squashed.